venia-integration-tests/src/tests/e2eTests/offline/offlineTest.spec.js

```diff
@@ -91,63 +79,58 @@ describe('PWA-1085: Verify cached pages are rendered correctly on offline mode',
             timeout: 60000
         });
 
-        // called to wait cache load properly
-        cy.wait(4000);
-
-        cy.visit(categoryDresses.url);
-
-        cy.wait(['@gqlGetCategoryDataCallQuery'], {
-            timeout: 60000
-        });
-
-        cy.wait(['@gqlGetCategoriesQuery'], {
-            timeout: 60000
-        });
+        cy.visit(categoryTops.url);
 
-        cy.wait(['@gqlGetStoreConfigDataForGallery'], {
-            timeout: 60000
-        });
-
-        cy.wait(4000);
+        cy.wait(
+            [
+                '@gqlGetCategoryDataCallQuery',
+                '@gqlGetStoreConfigDataForGallery'
+            ],
+            {
+                timeout: 60000
+            }
+        );
 
         triggerSearch();
+
         searchFromSearchBar(searchData.validSku1, false);
 
         cy.wait(['@gqlGetAutoCompleteResultsQuery'], {
             timeout: 60000
         });
 
-        cy.visit(categoryDresses.url);
-        cy.wait(4000);
+        assertProductIsInProductSuggestion(
+            searchData.validProductName1,
+            searchData.validProductHref1
+        );
 
-        cy.wait(['@gqlGetCategoriesQuery'], {
-            timeout: 60000
-        });
+        cy.visit(categoryTops.url).then(() => cy.wait(WAIT__TIME)); // cy.wait needed to assert that Cypress cached files
```

Comment:
Why do we need to wait?
Can't we check if the page contains something specific like a title or a list?